### PR TITLE
snapshot/backup docs improvement

### PR DESCRIPTION
**Pull request by jerrinot**
TODO:

- [x] figure out how to link it with reference doc at https://questdb.io/docs/reference/sql/snapshot/ - this should be improved too
- [ ] editorial changes

### CODE REVIEW

**Comment by jerrinot** [comment]
@goodroot it's ready for your pass. feel free to edit it as you see fit. the reference documentation could be more ... reference-like and the guide could use more imperative language. but I think this is still an improvement compared to the current state.

File: operations/backup.md [hunk 1]
```diff
@@ -6,55 +6,163 @@ description:
...
 
----
+### Terminology
 
-:::caution
+This guide uses the word "snapshot" in 2 different meanings:
+
+- **Database snapshot**: Instructs QuestDB to record the state of the database at a specific point in time. This is done
+  via `SNAPSHOT PREPARE` SQL command.
+- **Filesystem and volumne snapshot**: A point-in-time copy of the filesystem that can be used to create a backup. This is done
```
[the remark this answers is not in the document]
Comment (jerrinot) [reply operations/backup.md]: @nwoolmer thanks!

**Review by goodroot — APPROVED** [review]

**Review by goodroot — APPROVED** [review]

**Comment by goodroot** [comment]
Much appreciated, this is a large improvement.

Made a somewhat bold decision to deprecate the backup doc & redir to snapshot.

The main reason is to clear up the search/discovery issue.

https://github.com/questdb/questdb/issues/4811

> Use the deprecated BACKUP operation. Not only it's deprecated, it does not support incremental backups either.

Updated this line to point to a gist vs. the documentation.

However, is it also the case that `BACKUP` has been deprecated since 7.3.3?

Is it still available today in 8.1.0?

**Comment by jerrinot** [comment]
@goodroot: thanks improving this PR!

About the BACKUP situation: Between 7.3.3(?) it was deprecated on all platforms, but Windows. From 8.1 upwards it's deprecated on all platforms, including Windows. Yet the command itself is still there and works.

**Comment by goodroot** [comment]
@jerrinot Got it - thanks for confirming.

**Pull request merged**